src/tools/options.cpp

```diff
@@ -122,6 +122,16 @@ bool tools::parseConfigurationParameter(Configuration& config, const std::string
         config.useOpt = false;
         return true;
     }
+    if(arg == "--safe-stop")
+    {
+        config.stopWhenVerificationFailed = true;
+        return true;
+    }
+    if(arg == "--no-safe-stop")
```

Comment:
I don't really like the naming of the options, they don't really make clear, what they are used for.

Wouldn't something like `--ignore-verification-error` or `--verification-error` and `--no-verification-error` be better? These suggestions are also not great, but more accurate.

Reply:
I agree... I was thinking it is too long to use daily-bases, but these should be as you said.